RPerformanceTracking/Private/_RPTClassManipulator+WKWebView.m

```diff
@@ -254,10 +257,6 @@ + (void)_swizzleWKWebViewNavDelegate:(id<WKNavigationDelegate>)delegate
         {
             ((void(*)(id, SEL, id, id, id))originalImp)(selfRef, selector, webView, navigationResponse, decisionHandler);
         }
-        else
-        {
-            decisionHandler(WKNavigationResponsePolicyAllow);
-        }
 
```

Comment:
Yep this should crash if the swizzle gets called.
It'd be nice if the branch was rebased on master so only this diff is shown.